src/qml/components/ChatMessage.qml

```diff
@@ -159,10 +159,31 @@ Control {
 
                     Label {
                         Layout.maximumWidth: column.maxWidth
+                        visible: chatMessage.attachmentStatus == Enums.AttachmentStatus.Loaded
                         text: attachmentDisplaySize
                         color: "white"
                         font.pixelSize: UiHelper.fixFontSz(10)
                     }
+
+                    Label {
+                        Layout.maximumWidth: column.maxWidth
+                        visible: chatMessage.attachmentStatus == Enums.AttachmentStatus.Loading
+                        text: {
```

Comment:
Add new role to C++ model and calculate this text there.
You are looking for comma here. While it works for English locale, comma can be missing in others.

Reply:
Okay, done